README.md

```diff
@@ -112,6 +112,20 @@ It starts etcd and simulator-server locally.
 
 To run the frontend, please see [README.md](web/README.md) on ./web dir.
 
+## [Beta] Existing cluster importing
```

Comment:
As an aside, we should have said "external cluster", I think.
But I named that service as `replicateexistingcluster`...

Reply:
> we should have said "external cluster"

Tashikani, it seems better 😓 
You can rename it before releasing v0.1.0 if you like. But, it is up to you; not big problem here.